tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_spme.py

```diff
@@ -156,6 +156,33 @@ def test_well_posed_ec_reaction_limited(self):
         model.check_well_posedness()
 
 
+# class TestSPMeWithCrack(unittest.TestCase):
+#     def test_well_posed_none_crack(self):
```

Comment:
If Rob's fix sorted out the issue with this test, then uncommenting these lines (for all models) should get the coverage sorted out.